label_studio/ml_model_providers/migrations/0002_auto_20240718_1758.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.25 on 2024-07-18 17:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ml_model_providers', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='modelproviderconnection',
+            name='deployment_name',
+            field=models.CharField(blank=True, help_text='Azure OpenAI deployment name', max_length=512, null=True),
+        ),
+        migrations.AddField(
+            model_name='modelproviderconnection',
+            name='endpoint',
+            field=models.CharField(blank=True, help_text='Azure OpenAI endpoint', max_length=512, null=True),
+        ),
+        migrations.AlterField(
+            model_name='modelproviderconnection',
+            name='provider',
+            field=models.CharField(choices=[('OpenAI', 'OpenAI'), ('Azure', 'Azure')], default='OpenAI', max_length=255),
```

Comment:
Would changing that field require DB remigration?

Reply:
Yes it seems every new choice we add will cause this small migration